Selenium_Basics/EpamNavigationTests2.cs

```diff
@@ -0,0 +1,109 @@
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+
+namespace Selenium_Basics;
+
+public class EpamNavigationTests2
+{
+    private IWebDriver driver { get; set; }
+
+    [SetUp]
+    public void SetupBrowser()
+    {
+        var options = new ChromeOptions();
+        options.BinaryLocation = "/Applications/Google Chrome.app/Contents/MacOS/Google Chrome";
+        driver = new ChromeDriver(options);
+    }
+    
+    [TearDown]
+    public void TearDown()
+    {
+        driver.Quit();
+    }
+    
+    [Test]
+    public void CheckThatListOfCountriesContainsAmerEMEAandAPAC()
+    {
+        var epamUrl = "https://www.epam.com/";
+
+        driver.Manage().Window.Maximize();
+        driver.Navigate().GoToUrl(epamUrl);
+        driver.FindElement(By.XPath("//*[text()='Careers']//ancestor::*[@class='top-navigation__item-text']")).Click();
```

Comment:
I suppose we discussed this locator in Skype and I asked you to rewrite it as we should not search by text

Reply:
by href it's not clicking the right element